lldb/test/API/functionalities/postmortem/elf-core/TestLinuxCore.py

```diff
@@ -833,6 +845,107 @@ def test_riscv64_regs_gpr_only(self):
             substrs=["registers were unavailable"],
         )
 
+    @skipIfLLVMTargetMissing("LoongArch")
+    def test_loongarch64_regs(self):
+        # check registers using 64 bit LoongArch64 core file containing GP-registers only
```

Comment:
Maybe this is a terminology difference. To me GP register would mean "general purpose" and not include floating point registers. But perhaps you always have an FPU so they are kinda general in that they always exist.

I'd just change it to "GP and FP registers" to prevent any confusion.

Reply:
There is confusion, I will revise it later.

Reply:
There is confusion, I will revise it later.